examples/bare-metal/explorer_board/src/audio_pipeline/audio_pipeline.c

```diff
@@ -5,48 +5,50 @@
 #include <platform.h>
 #include <xs1.h>
 #include <string.h>
-#include <xcore/assert.h>
-#include <xcore/chanend.h>
-#include <xcore/channel.h>
-#include <xcore/channel_streaming.h>
-#include <xcore/parallel.h>
-#include <xcore/port.h>
-#include <xcore/hwtimer.h>
 #include <xcore/triggerable.h>
 
 /* SDK headers */
-#include "soc.h"
 #include "xcore_utils.h"
 #include "mic_array.h"
+#include "bfp_math.h"
 
 /* App headers */
-#include "app_conf.h"
+#include "../app_conf.h"
```

Comment:
Could you return this to a non relative path, as it will prevent portability by forcing a specific file structure.